scripts/core/EXT.rst

```diff
@@ -0,0 +1,108 @@
+  - All structures, enumerations and other types must follow `${x}_<name>_ext_t` snake case convention
+  - All enumerator values must use all caps `${X}_ENUM_EXT_ETOR_NAME` convention
+  - All handle types must end with `ext_handle_t`
+  - All descriptor structures must end with `ext_desc_t`
+  - All property structures must end with `ext_properties_t`
+  - All flag enumerations must end with `ext_flags_t`
+## --validate=on
+
+The following naming conventions must be followed for **experimental** extensions:
+
+## --validate=off
+  - The extension name must begin with `${x}_exp`
+  - Experimental extensions may be added and removed from the driver at any time.
+  - Experimental extensions are not guaranteed to be forward- or backward-compatible between versions.
+  - Experimental extensions are not guaranteed to be supported in production driver releases; and may appear and disappear from release to release.
+  - All extension functions must be postfixed with `Exp`
+  - All macros must use all caps `${X}_<NAME>_EXP` convention
+  - All structures, enumerations and other types must follow `${x}_<name>_exp_t` snake case convention
+  - All enumerator values must use all caps `${X}_ENUM_EXP_ETOR_NAME` convention
+  - All handle types must end with `exp_handle_t`
+  - All descriptor structures must end with `exp_desc_t`
+  - All property structures must end with `exp_properties_t`
+  - All flag enumerations must end with `exp_flags_t`
+## --validate=on
+
+Extending Enumerations
+----------------------
+
+Any existing enumeration may be extended by adding new enumerators. Enumerators must use the extensions naming 
+convention and values should be assigned to avoid future compatibility issues.
+
+
+Extending Structures
+--------------------
+
+Any structure derived from `${x}_base_desc_t`` or `${x}_base_properties_t`` can be extended using a structure chain
+by adding a pointer to the extended structure in `pNext` member variable. No other method of extending structures is allowed.
+
+A structure chain can contain more than one extension structure, in any order. Therefore, extensions should not
+be dependent on their order relative to other extensions and the implementation must be order agnostic. In addition,
+the implementation will ignore extended structures that it does not support.
+
+The extension must document the specific structures and functions that may be extended using the structure chain.
+
+Adding an extension
+===================
+
+* Extend the specification with the functions, enumerations, macros, and structures required by the extension, ensuring that
+  all additions fully comply with the naming and ABI requirements detailed above.
+
+* Document the extension in `EXT_<ext-name>.rst` file based on the `EXT_Template.rst`, ensuring it is added to the list of
+  extensions below.
+
+* The extension will only be accepted if all conditions are met and its addition is agreed upon by the Working Group.
+
+
+List of Standard Extensions
```

Comment:
Just a thought, maybe these lists should also track the implementation status of the extensions, so whether they are implemented, and then which version of UR supports which version of the extension, particularly for the experimental extensions which may change, so high-level languages know what they can use when updating the UR interface.